src/chat/chat/study-chat.service.ts

```diff
@@ -54,14 +56,16 @@ export class StudyChatService {
       membersIds,
     );
     await this.usersService.addMessage(membersIds, newMessage._id.toString());
+    await this.roomsService.addMessage(message.room, newMessage._id);
     const response: MessageResponseDto = {
       date: newMessage.created,
       email: user.email,
       file: this.toDto.toFile(newMessage.file),
       id: newMessage._id.toString(),
-      photo: user.photo,
+      photo: 'http://localhost:3000/images/' + user.photo,
```

Comment:
move string "http://localhost:3000/" to .env variable and use this variable in all places